circleci.tf

```diff
@@ -36,14 +36,44 @@ variable "services_instance_type" {
 
 variable "builder_instance_type" {
     description = "instance type for the builder machines.  We recommend a r3 instance"
-    default = "r3.2xlarge"
+    default = "r3.4xlarge"
+}
+
+variable "circle_container_cpus" {
+    description = "CPUs allocated per container"
+    default = "2"
+}
+
+variable "circle_container_image_id" {
+    description = "Container image ID"
+    default = "circleci-precise-container_0.0.1338"
```

Comment:
Matches the current default in https://s3.amazonaws.com/circleci-enterprise/init-builder-0.2.sh